Firestore/Protos/nanopb_cpp_generator.py

```diff
@@ -246,16 +329,308 @@ def nanopb_write(results):
   response = plugin_pb2.CodeGeneratorResponse()
 
   for result in results:
-    f = response.file.add()
-    f.name = result['headername']
-    f.content = result['headerdata']
+    base_filename = result['headername'].replace('.nanopb.h', '')
+    file_printers = pretty_printing_info[base_filename]
```

Comment:
Isn't the thing returned here the `FilePrettyPrintingInfo` instance for the file? Shouldn't this be `file_printer`?